Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.18
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update
+RUN apt-get install -y curl git
```

Comment:
Sorry last update and then I promise that I will merge in this pull request.  So I do not think that we need these two `RUN apt-get` lines anymore so maybe better to remove them and keep the Dockerfile clean and simple.  That should probably do it.

Reply:
@jbvandefliert Good call, updated